_commands/toys/cowsay.md

```diff
@@ -2,23 +2,111 @@
 ---
 
 cowsay
--------
+--
+`cowsay` generates an ASCII picture of a cow saying something provided by  the  user.
 
-ToDo: Needs Completion
 
 ~~~ bash
-*short* example of command usage and output
+$ cowsay Hi There
+ __________ 
+< Hi There >
+ ---------- 
+        \   ^__^
+         \  (oo)\_______
+            (__)\       )\/\
+                ||----w |
+                ||     ||
 ~~~
 
 <!--more-->
 
+
+
 ### Useful Options / Examples
+There are many useful options for this intersting cowsay command.
+The complete list is: cowsay  `[-e  eye_string]` `[-f cowfile]` `[-h]` `[-l]` `[-n]` `[-T tongue_string]` `[-W column]` `[-bdgpstwy]`
+
+`-e` allows the user to customize the cow's eyes which are defaulted to "oo" as seen above.
+`-T` allows the user to add a tongue string to the cow image. By default there is no tongue string.
+`-W` allows the user to customize the number of character printed in the message before the message wraps around
+`-[-bdgpstwy]` are all options to define different pre-made cows emotional states. For example, adding the -g flag will result in a greedy cow to print and -s will result in a stoned cow to print.
 
-#### Example command
+#### cowsay [-bdgpstwy] [`user_input`]
+
+~~~ bash
+$ cowsay -s hi there
+ __________ 
+< hi there >
```

Comment:
I think line 37 was supposed to have a `-T` instead of an `-s`, since the tongue is sticking out = P